pages/create/index.js

```diff
@@ -0,0 +1,311 @@
+    //     : currentPet + 1
+    // );
+
+    let newPetIndex;
+    direction === "back" &&
+      currentPet === 0 &&
+      (newPetIndex = animalList.length - 1);
+    direction === "back" && currentPet !== 0 && (newPetIndex = currentPet - 1);
+    direction !== "back" &&
+      currentPet === animalList.length - 1 &&
+      (newPetIndex = 0);
+    direction !== "back" &&
+      currentPet !== animalList.length - 1 &&
+      (newPetIndex = currentPet + 1);
+    setCurrentPet(newPetIndex);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    const petData = {
+      ...data,
+      type: animalList[currentPet].type,
+      picture: animalList[currentPet].icon,
+    };
+    event.target.reset();
+    setCharacteristics({
+      characteristic1: "",
+      characteristic2: "",
+    });
+    onCreatePet(petData);
+  }
+
+  return (
+    <StyledCreatePage>
+      <StyledForm onSubmit={handleSubmit}>
+        <PetSelection
+          onChangePet={handleChangePet}
+          animalList={animalList}
+          currentPet={currentPet}
+          name="type"
+        />
+        <StyledMainField>
+          <StyledFormElement>
+            <label htmlFor="type">Type:</label>
+            <input
+              name="type"
+              id="type"
+              value={animalList[currentPet].type}
+              disabled
+            />
+          </StyledFormElement>
+          <StyledFormElement>
+            <label htmlFor="name">Name:</label>
+            <input
+              name="name"
+              id="name"
+              placeholder="Samantha"
+              maxLength={30}
+              required
+            />
+          </StyledFormElement>
+          <StyledFormElement>
+            <label htmlFor="characteristic">Characteristics:</label>
+            <StyledCharacteristicsContainer>
+              <select
+                name="characteristic1"
+                id="characteristic"
+                value={characteristics.characteristic1}
+                onChange={(event) =>
+                  setCharacteristics({
+                    ...characteristics,
+                    characteristic1: event.target.value,
+                  })
+                }
+                required
+              >
+                <option value="" disabled>
+                  Choose Nr. 1
+                </option>
+                {characteristicOptions.map((option, index) => (
+                  <option
+                    key={index}
+                    disabled={
+                      characteristics.characteristic2 ===
+                        option.characteristic ||
+                      characteristics.characteristic2 === option.opposite
+                    }
+                  >
+                    {option.characteristic}
+                  </option>
+                ))}
+              </select>
+              <select
```

Comment:
this form element does not have a label it is connected to, making the form less accessible